content/asset-tracking/index.textile

```diff
@@ -0,0 +1,157 @@
+
+h3(#default-resolution-constraints). Default resolution constraints
+
+The Ably-provided default resolution constraints consist of:
+
+- @DefaultResolutionSet@ := The set of resolutions which must be defined in order to populate the @DefaultResolutionConstraints@.
+- @DefaultProximity@ := This is a distance to destination at which resolution of updates is increased to improve user experience.
+- @batteryLevelThreshold@ := This is a level of battery on the device that runs Publishing SDK that is considered low. When the battery level falls below this, the resolution of updates is reduced. Default value: 10%. In the range 0.0f (no battery) to 100.0f (full battery).
+- @lowBatteryMultiplier@ := The multipler to be applied to the interval when the battery level is below @batteryLevelThreshold@.
+
+The following example shows how the default resolution constraints are populated:
+
+```[android]
+// Prepare Resolution Constraints for an asset that will be used in the Resolution Policy
+val exampleConstraints = DefaultResolutionConstraints(
+    DefaultResolutionSet( // this constructor provides one Resolution for all states
+        Resolution(
+            accuracy = Accuracy.BALANCED,
+            desiredInterval = 1000L,
+            minimumDisplacement = 1.0
+        )
+    ),
+    proximityThreshold = DefaultProximity(spatial = 1.0),
+    batteryLevelThreshold = 10.0f,
+    lowBatteryMultiplier = 2.0f
+)
+```
+
+h2(#resolution-policy). Resolution policy
+
+Resolution policy defines the strategy by which the various @ResolutionRequests@ and preferences are translated by @Publisher@ instances into a target @Resolution@.
+
+As different use cases have different requirements for battery optimization, Ably Asset Tracking SDKs provide a flexible @ResolutionPolicy@ interface, that allows SDK users to provide their own implementation of resolution optimization logic. Ably also provides a "default resolution policy":#default-resolution-policy, which is suitable for common use cases.
+
+h3(#default-resolution-policy). Default resolution policy
+
+Ably Asset Tracking SDKs come with a @DefaultResolutionPolicy@ implementation provided out of the box, which is suitable for common use cases.
+
+Each @Trackable@ added to the @Publisher@ has a destination and set of @ResolutionConstraints@. Ably provides a set of "default resolution constraints":#default-resolution-constraints, @DefaultResolutionContraints@.
+
+When the SDK is running, on each change of the device or trackable status, the @ResolutionPolicy@ recalculates which resolution should be applied. Example changes include: 
+
+* Subscribers added or removed from the list of subscribers.
+* Battery or proximity threshold hit.
+* Subscriber closes app.
+
+The following logic is used:
+
+1. When comparing resolutions, the more precise resolution is always selected.
```

Comment:
To be more precise we are comparing each field of the resolution (accuracy, desiredInterval, minimumDisplacement) and we're keeping the most precise value for each of those fields. So the final resolution could be a hybrid of all the provided resolutions. 
For example, it can have the accuracy of resolution A, the desiredInterval of resolution B and the minimumDisplacement of resolution C :wink:

Reply:
Useful info added. Fixed.